aws/resource_aws_ses_receipt_rule_set_test.go

```diff
@@ -2,15 +2,84 @@ package aws
 
 import (
 	"fmt"
+	"log"
 	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
 )
 
+func init() {
+	resource.AddTestSweepers("aws_ses_receipt_rule_set", &resource.Sweeper{
+		Name: "aws_ses_receipt_rule_set",
+		F:    testSweepSesReceiptRuleSets,
+	})
+}
+
+func testSweepSesReceiptRuleSets(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %w", err)
+	}
+	conn := client.(*AWSClient).sesconn
+
+	// You cannot delete the receipt rule set that is currently active.
+	// Setting the name of the receipt rule set to make active to null disables all email receiving.
+	log.Printf("[INFO] Disabling any currently active SES Receipt Rule Set")
+	_, err = conn.SetActiveReceiptRuleSet(&ses.SetActiveReceiptRuleSetInput{})
+	if testSweepSkipSweepError(err) {
```

Comment:
GovCloud currently fails on this operation with:

```
2020/05/11 15:37:01 [INFO] Disabling any currently active SES Receipt Rule Set
2020/05/11 15:37:02 [ERROR] Error running Sweeper (aws_ses_receipt_rule_set) in region (us-gov-west-1): error disabling any currently active SES Receipt Rule Set: InvalidAction: Unavailable Operation
```

That error seems specific enough that we might want to add `isAWSErr(err, "InvalidAction", "Unavailable Operation")` to `testSweepSkipSweepError()` or otherwise `|| isAWSErr(err, "InvalidAction", "Unavailable Operation")` here. 👍

Reply:
I'll add to `testSweepSkipSweepError()`.

Reply:
Done.